website/docs/components/alert/partials/guidelines/guidelines.md

```diff
@@ -56,59 +56,59 @@ Use color logically.
 - **Neutral** to provide general information to the user regarding the current context or relevant actions.
 - **Highlight** to provide general or promotional information to the user prominently.
 - **Success** to indicate a successful action.
-    - Use the success variant sparingly. To communicate success after a user action is performed, use [Toast](/components/toast).
+  - Use the success variant sparingly. To communicate success after a user action is performed, use [Toast](/components/toast).
 - **Warning** to help users avoid an issue. Provide guidance and actions, if possible.
 - **Critical** to indicate critical errors that need immediate action.
 
 ### Critical alerts
 
-Use critical alerts to convey crucial information or errors, such as validation errors that require user attention or action. 
+Use critical alerts to convey crucial information or errors, such as validation errors that require user attention or action.
 
 !!! Do
 
 Use the Alert for more intrusive message communication about errors or critical disruptions at an application, page, or section level where users need to take immediate action.
 
 <Hds::Alert @type="inline" @color="critical" as |A|>
-    <A.Title>You have exceeded 50 applies this month</A.Title>
-    <A.Description>You may only invoke applies that destroy managed resources. Upgrade now and access additional product features, unlimited applies, and increased concurrency.</A.Description>
-    <A.Button @text="Upgrade" @color="secondary" @onClick={{this.noop}} />
-    <A.Link::Standalone @color="secondary" @icon="arrow-right" @iconPosition="trailing" @text="View usage" @href="#" />
-  </Hds::Alert>
+  <A.Title>You have exceeded 50 applies this month</A.Title>
```

Comment:
@jorytindall can you revert all these reformatting changes, to reduce the noise? thanks

Reply:
Is there an "easy" way to do that? I could use a little assistance there, it was likely included in a commit that had other changes as a part of it.

Reply:
I've pushed a commit that undoes these changes (only these, don't worry 😀).